ports/psoc6/boards/CY8CPROTO-063-BLE/manifest.py

```diff
@@ -1,3 +1,4 @@
 freeze("$(PORT_DIR)/freeze")
 include("$(MPY_DIR)/extmod/asyncio")
 require("mip")
+require("neopixel")
```

Comment:
we should only add this to the demo branch, but I would not by default freeze application specific libraries.